src/routes/invidious_routes/download.ts

```diff
@@ -73,6 +73,7 @@ export default function getDownloadHandler(app: Hono) {
 
             const urlQueriesForLatestVersion = new URLSearchParams();
             urlQueriesForLatestVersion.set("id", videoId);
+            urlQueriesForLatestVersion.set("check", check || "");
```

Comment:
What's that about?

Reply:
it's for request verification. The implementation of Downloads used the `latestVersion` route within the `download` route, so we need to add the check from the call to `download` to the `latestVersion`, otherwise it fails due to a missing check if you have `verify_requests` on